src/main/java/it/gov/pagopa/timeline/constants/TimelineConstants.java

```diff
@@ -3,6 +3,9 @@
 public class TimelineConstants {
 
   public static final String ERROR_MANDATORY_FIELD = "The field is mandatory!";
+  public static final String AUTHORIZED = "AUTHORIZED";
+  public static final String REWARDED = "REWARDED";
```

Comment:
Let's call the status constants with an explicit prefix (e.g. TRX_STATUS_REWARDED) so that we can say for what they're used

```suggestion
  public static final String TRX_STATUS_REWARDED = "REWARDED";
```